plugins/ILibPlugin/index.js

```diff
@@ -169,11 +169,12 @@ class ILibPlugin {
 				if (pkgName.indexOf('@enact') === 0) {
 					this.options.create = false;
 				}
-				if (pkgName === '@enact/i18n') {
-					this.options.ilib = 'ilib';
-				} else {
-					this.options.ilib = packageSearch(process.cwd(), '@enact/i18n/ilib');
-				}
+				// look for ilib as a root-level node_module package location
+				this.options.ilib =
+					// Backward compatability for old Enact libraries
+					packageSearch(process.cwd(), path.join('@enact', 'i18n', 'ilib')) ||
+					packageSearch(process.cwd(), 'ilib') ||
+					(pkgName === '@enact/i18n' && fs.existsSync(path.join(process.cwd(), 'ilib')) && 'ilib');
```

Comment:
Also, can you explain what the last part is for?  Does that find a root `ilib` folder? If so, how does that work with the fallbacks below that seem to assume `node_modules`?

Reply:
The last line is for potential compiles/webpack usage within i18n library itself. Extreme niche, but it affected us during the Karma day when we compiled tests.